package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "restaurant-js",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
@balerum03, Good job so far 👍 

This project came after Webpack lesson, we need to use the knowledge in this project.

According to the pathwright, you need to add scripts that are necessary to build your project in package.json.

![image](https://user-images.githubusercontent.com/31897434/105122773-2333d500-5ad7-11eb-964a-567f5ceadc18.png)

https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635163/



_Heads-Up:_ you're also missing `webpack-dev-server` in `devDependencies`